Jenkinsfile

```diff
@@ -4,7 +4,7 @@ pipeline {
   stages {
     stage("Test Build") {
       when {
-        not { anyOf { branch 'staging' ; branch 'xenial' ; branch 'trusty  ' ; buildingTag() } }
```

Comment:
Should we include `focal` in this list?

Reply:
yes< i intended to but seems it got missed. thanks for catching this.